rel/overlay/etc/default.ini

```diff
@@ -141,12 +141,16 @@ max_db_number_for_dbs_info_req = 100
 ; admin_only_all_dbs = true
 
 ;[jwt_auth]
-; Symmetric secret to be used when checking JWT token signatures
-; secret =
 ; List of claims to validate
 ; required_claims = exp
 ; List of algorithms to accept during checks
 ; allowed_algorithms = HS256
+;
+; [jwt_keys]
+; Default secret to be used when checking JWT token signatures
```

Comment:
Maybe just add a note here for people like me who only want to configure a single symmetric like "put your single symmetric key here"?

Reply:
I'll expand the text. I also think the symmetric key will need to be base64 encoded in the config, so we can allow binary values. So far we've assumed text is ok.

Reply:
I've addressed both things, please see if the updated text is more helpful.

Reply:
Hmm, not sure how I feel about base64'ing the key in config. I've yet to encounter any configuration format that would insist on this and it could frustrate deployments for some people.